### PR TITLE
EV-25: Update Event Card

### PR DESCRIPTION
**Pull request by LisovyiNazar**
## Jira
*[EV-25: Update Event Card](https://chita.atlassian.net/browse/EV-25?atlOrigin=eyJpIjoiMDVhZjBhNDdjNmQ4NDYzNDk1ZDg1NDViMWZkY2VhOTkiLCJwIjoiaiJ9)

## Summary of issue

Changed design of EventCard
Changed design of EventListCard
Changed design of SliderForEvents
Maked SliderForEvents adaptive

## CHECK LIST
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by MelenkoOleksandr — CHANGES_REQUESTED** [review]
EventlListCard.scss - Put fonts in fonts.scss file and then import them
SliderForEvents.scss - something wrong with import

File: src/components/landing/EventsListItem/EventListCard/EventListCard.scss [hunk 1]
```diff
@@ -33,8 +30,6 @@
         position: absolute;
         box-shadow: 10px 10px 40px 0 rgba(0, 0, 0, 0.75);
```
Comment (MelenkoOleksandr) [inline src/components/landing/EventsListItem/EventListCard/EventListCard.scss]: use constants))

File: src/components/landing/EventCard/EventCard.js [hunk 2]
```diff
@@ -1,31 +1,62 @@
...
+      <div className="card-image-container">
+        <img className="card-image" src={photo} alt="" />
       </div>
-      <IconsEventCard styleForIcon={sliderOrMatrix} />
+      <div className="card-content">
+        <div className="card-date">
+          <div className="date-container">
+            <span className="day">{moment(dateFrom).format("DD")}</span>
+          </div>
+        </div>
```
Comment (ghost) [inline src/components/landing/EventCard/EventCard.js]: if block has only 1 child, u don`t need extra wrapper
we discussed it

File: src/components/landing/EventCard/EventCard.scss [hunk 3]
```diff
@@ -1,29 +1,103 @@
...
-    border-radius: 20px;
+  width: 300px;
+  margin: 15px;
+  box-shadow: 10px 10px 22px 0 rgba(0, 0, 0, 0.75);
+  background-color: $white;
+  .card-image-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
```
Comment (ghost) [inline src/components/landing/EventCard/EventCard.scss]: use constants

File: src/components/landing/EventCard/EventCard.scss [hunk 4]
```diff
@@ -1,29 +1,103 @@
...
-  &__poster {
-    border-radius: 20px;
+  width: 300px;
+  margin: 15px;
+  box-shadow: 10px 10px 22px 0 rgba(0, 0, 0, 0.75);
+  background-color: $white;
+  .card-image-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
```
Comment (ghost) [inline src/components/landing/EventCard/EventCard.scss]: use mixin

File: src/components/landing/EventCard/EventCard.scss [hunk 5]
```diff
@@ -1,29 +1,103 @@
...
+    display: flex;
+    .card-date {
+      width: 150px;
+      position: relative;
+      padding: $minPadding;
+      .date-container {
+        height: 100%;
+        display: flex;
+        justify-content: center;
+        align-items: center;
```
Comment (ghost) [inline src/components/landing/EventCard/EventCard.scss]: use mixin

File: src/components/landing/EventCard/EventCard.scss [hunk 6]
```diff
@@ -1,29 +1,103 @@
...
+        .day {
+          display: flex;
+          justify-content: center;
+          align-items: center;
```
Comment (ghost) [inline src/components/landing/EventCard/EventCard.scss]: use mixin

File: src/components/landing/EventCard/EventCard.scss [hunk 7]
```diff
@@ -1,29 +1,103 @@
...
+          color: $lightThemeAccept;
+          font-family: 'Robinson', sans-serif;
+          height: 100px;
+          text-shadow: 0 0 20px $white;
+          margin-bottom: 15px;
+          font-weight: bold;
+          font-size: 80px;
```
Comment (ghost) [inline src/components/landing/EventCard/EventCard.scss]: use constant or mixin

File: src/components/landing/EventCard/EventCard.scss [hunk 8]
```diff
@@ -1,29 +1,103 @@
...
+      width: 100%;
+      display: flex;
+      flex-direction: column;
+      justify-content: space-between;
+      .card-info-header {
+        padding: $minPadding;
+        height: 100%;
+        text-transform: uppercase;
+        font-weight: bold;
+        font-size: 14px;
```
Comment (ghost) [inline src/components/landing/EventCard/EventCard.scss]: use constants

File: src/components/landing/EventCard/EventCard.scss [hunk 9]
```diff
@@ -1,29 +1,103 @@
...
+        color: $black;
+        align-items: center;
+      }
+      .card-buttons {
+        padding: $minPadding;
+        height: 40px;
+        display: flex;
+        justify-content: flex-end;
+        align-items: center;
```
Comment (ghost) [inline src/components/landing/EventCard/EventCard.scss]: use mixin

File: src/components/landing/EventCard/EventCard.scss [hunk 10]
```diff
@@ -1,29 +1,103 @@
...
+        .like, .more {
+          display: flex;
+          justify-content: center;
+          align-items: center;
```
Comment (ghost) [inline src/components/landing/EventCard/EventCard.scss]: use mixin

File: src/components/landing/EventCard/EventCard.scss [hunk 11]
```diff
@@ -1,29 +1,103 @@
...
-    line-height: 127.9%;
-    color: $white;
+  .month {
+    padding: $minPadding;
+    color: $lightThemeAccept;
+    font-family: 'Robinson', sans-serif;
+    height: 40px;
+    text-transform: uppercase;
+    font-size: 45px;
+    width: 100%;
```
Comment (ghost) [inline src/components/landing/EventCard/EventCard.scss]: use constant

File: src/components/landing/EventsListItem/EventListCard/EventListCard.scss [hunk 12]
```diff
@@ -1,47 +1,261 @@
...
-  & img {
-    flex-basis: 30%;
-    border-radius: $defaultBorderRadius;
+  justify-content: center;
+  .card__item {
+    height: 280px;
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
```
Comment (ghost) [inline src/components/landing/EventsListItem/EventListCard/EventListCard.scss]: use mixin

File: src/components/landing/EventsListItem/EventListCard/EventListCard.scss [hunk 13]
```diff
@@ -1,47 +1,261 @@
...
+          text-transform: uppercase;
+          font-weight: bold;
+          font-size: 18px;
+          color: $lightThemeAccept;
+          position: relative;
+        }
+        .card__item-ovner {
+          display: flex;
+          justify-content: center;
+          align-items: center;
```
Comment (ghost) [inline src/components/landing/EventsListItem/EventListCard/EventListCard.scss]: use mixin

File: src/components/landing/EventsListItem/EventListCard/EventListCard.scss [hunk 14]
```diff
@@ -1,47 +1,261 @@
...
+          font-size: 14px;
+          color: #848484;
+          padding-bottom: 15px;
+        }
+      }
+    }
+    .card__item_bottom-row {
+      display: flex;
+      justify-content: flex-end;
+      align-items: center;
```
Comment (ghost) [inline src/components/landing/EventsListItem/EventListCard/EventListCard.scss]: use mixin

File: src/components/landing/EventsListItem/EventListCard/EventListCard.scss [hunk 15]
```diff
@@ -1,47 +1,261 @@
...
+        .day, .month {
+          color: $lightThemeAccept;
+          font-family: 'Robinson', sans-serif;
+          height: 40px;
+        }
+        .day {
+          text-shadow: 0 0 20px $white;
+          margin-bottom: 15px;
+          font-weight: bold;
+          font-size: 100px;
```
Comment (ghost) [inline src/components/landing/EventsListItem/EventListCard/EventListCard.scss]: use constants or mixin

File: src/components/landing/EventsListItem/EventListCard/EventListCard.scss [hunk 16]
```diff
@@ -1,47 +1,261 @@
...
+          line-height: 30px;
+        }
+        .month {
+          text-transform: uppercase;
+          font-size: 45px;
```
Comment (ghost) [inline src/components/landing/EventsListItem/EventListCard/EventListCard.scss:100]: use constants

File: src/components/landing/EventsListItem/EventListCard/EventListCard.scss [hunk 17]
```diff
@@ -1,47 +1,261 @@
...
+          .card-image {
+            min-width: 280px;
+            max-height: 150px;
+          }
+        }
+        .card__item_content {    
+          height: 200px;
+          justify-content: flex-start;
+          .card__item_header {
+            font-size: 13px;
```
Comment (ghost) [inline src/components/landing/EventsListItem/EventListCard/EventListCard.scss]: use constants

File: src/components/landing/EventsListItem/EventListCard/EventListCard.scss [hunk 18]
```diff
@@ -1,47 +1,261 @@
...
+          .day, .month {
+            color: $lightThemeAccept;
+            font-family: 'Robinson', sans-serif;
+            height: 40px;
+          }
+          .day {
+            text-shadow: 0 0 20px $white;
+            margin-bottom: 10px;
+            font-weight: bold;
+            font-size: 80px;
```
Comment (ghost) [inline src/components/landing/EventsListItem/EventListCard/EventListCard.scss]: use constants

File: src/components/landing/SliderForEvents/settingsForSlider.js [hunk 19]
```diff
@@ -1,25 +1,25 @@
...
-        centerMode: true,
       },
     },
     {
-      breakpoint: 480,
-      settings: {
-        arrows: false,
-        centerMode: true,
-      },
+      breakpoint: 1180,
```
Comment (ghost) [inline src/components/landing/SliderForEvents/settingsForSlider.js:21]: magic number

File: src/components/landing/SliderForEvents/settingsForSlider.js [hunk 20]
```diff
@@ -1,25 +1,25 @@
...
+  slidesToScroll: 3,
+  swapToSlide: true,
+  arrows: true,
   centerPadding: "0px",
-  dots: true,
-
   responsive: [
     {
-      breakpoint: 768,
+      breakpoint: 767,
```
Comment (ghost) [inline src/components/landing/SliderForEvents/settingsForSlider.js:12]: magic number

File: src/components/landing/SliderForEvents/settingsForSlider.js [hunk 21]
```diff
@@ -1,25 +1,25 @@
 const settingsForSlider = {
-  centerMode: true,
-  infinite: true,
-  slidesToShow: 3,
+  dots: true,
+  infinite: false,
   speed: 500,
```
Comment (StodolaNazariy) [inline src/components/landing/SliderForEvents/settingsForSlider.js:4]: magic number

File: src/actions/event/event-list-action.js [hunk 22]
```diff
@@ -74,7 +74,7 @@ export function getUpcomingEvents(filters) {
     // const jsonRes = await response.json();
     const jsonRes = await response.json();
```
Comment (andrii-kuzmichov) [inline src/actions/event/event-list-action.js:75]: rename to data

File: src/components/landing/EventCard/EventCard.js [hunk 23]
```diff
@@ -1,31 +1,78 @@
...
+    "April",
+    "May",
+    "June",
+    "July",
+    "August",
+    "September",
+    "October",
+    "November",
+    "December",
+  ];
```
Comment (andrii-kuzmichov) [inline src/components/landing/EventCard/EventCard.js]: Move to constants

File: src/components/landing/EventCard/EventCard.js [hunk 24]
```diff
@@ -1,31 +1,78 @@
...
+  const monthIndex = moment(dateFrom).format("M") - 1;
```
Comment (andrii-kuzmichov) [inline src/components/landing/EventCard/EventCard.js]: Move `M` to constatns as format date obj

File: src/components/landing/EventCard/EventCard.js [hunk 25]
```diff
@@ -1,31 +1,78 @@
...
+  const photoService = new PhotoService();
+  const [eventImage, setEventImage] = useState(eventDefaultImage);
+
+  useEffect(() => {
+    photoService.getPreviewEventPhoto(id).then(eventPreviewImage => {
+      if (eventPreviewImage) {
+        const image = URL.createObjectURL(eventPreviewImage);
+        setEventImage(image);
+      }
+    });
```
Comment (andrii-kuzmichov) [inline src/components/landing/EventCard/EventCard.js:23]: move to service

File: src/components/landing/EventCard/EventCard.js [hunk 26]
```diff
@@ -1,31 +1,78 @@
...
-      <img className="event-card__poster" src={photo} alt="Poster" />
-      <div className="event-card__inner-container">
-        <p className="event-card__inner-text">{title}</p>
+      <img
+        className="card-image-container card-image"
+        src={eventImage}
+        alt=""
+      />
+      <div className="card-content">
+        <div className="card-date date-container">
```
Comment (andrii-kuzmichov) [inline src/components/landing/EventCard/EventCard.js:37]: extra div

File: src/components/landing/EventCard/EventCard.js [hunk 27]
```diff
@@ -1,31 +1,78 @@
...
+          <span className="day">{moment(dateFrom).format("DD")}</span>
```
Comment (andrii-kuzmichov) [inline src/components/landing/EventCard/EventCard.js]: move `DD` to format date constants

File: src/components/landing/EventCard/EventCard.js [hunk 28]
```diff
@@ -1,31 +1,78 @@
...
+        </div>
+        <div className="card-info">
+          <span className="card-info-header">
+            {title.length > 35 ? `${title.slice(0, 35)}...` : title}
```
Comment (andrii-kuzmichov) [inline src/components/landing/EventCard/EventCard.js:44]: Strange logic move to utils

File: src/components/landing/EventCard/EventCard.js [hunk 29]
```diff
@@ -1,31 +1,78 @@
...
+          </span>
+          <div className="card-buttons">
+            <IconsEventCard className="like" styleForIcon="list" />
+            <NavLink to={`/home/events/${id}`} className="more bttn">
+              <BiRightArrowAlt size={30} />
```
Comment (andrii-kuzmichov) [inline src/components/landing/EventCard/EventCard.js:49]: `30` move to constants

File: src/components/landing/EventCard/EventCard.js [hunk 30]
```diff
@@ -1,31 +1,78 @@
...
+            </NavLink>
+          </div>
+        </div>
       </div>
-      <IconsEventCard styleForIcon={sliderOrMatrix} />
+      <span className="month">{monthes[monthIndex]}</span>
```
Comment (andrii-kuzmichov) [inline src/components/landing/EventCard/EventCard.js]: Refactoring logic

File: src/components/landing/EventCard/EventCard.scss [hunk 31]
```diff
@@ -1,29 +1,88 @@
...
-  position: relative;
-  display: inline-block;
-  max-width: 350px;
-  margin: 10px;
-
-  &__poster {
-    border-radius: 20px;
+  width: $distance300px;
+  margin: 15px;
+  box-shadow: $distance10px $distance10px $distance20px 0 $lightThemeHeader;
```
Comment (andrii-kuzmichov) [inline src/components/landing/EventCard/EventCard.scss]: remove specific for distance and create or reuse for box-shadow

File: src/components/landing/EventCard/EventCard.scss [hunk 32]
```diff
@@ -1,29 +1,88 @@
...
+      @include flex(row, center, center);
+      height: $percent100;
+      .day {
+        @include flex(row, center, center);
+        color: $lightThemeAccept;
+        font-family: 'Robinson', sans-serif;
+        height: $distance100px;
+        text-shadow: 0 0 $distance20px $white;
+        margin-bottom: $distance15px;
+        font-weight: bold;
```
Comment (andrii-kuzmichov) [inline src/components/landing/EventCard/EventCard.scss]: constants

File: src/components/landing/EventCard/EventCard.scss [hunk 33]
```diff
@@ -1,29 +1,88 @@
...
+      @include flex(column, start, space-between);
+      padding: $minPadding;
+      width: $percent100;
+      .card-info-header {
+        @include flex(row, center, center);
+        width: $percent100;
+        padding: $minPadding;
+        height: $distance100px;
+        text-transform: uppercase;
+        font-weight: bold;
```
Comment (andrii-kuzmichov) [inline src/components/landing/EventCard/EventCard.scss]: the same

File: src/components/landing/EventCard/EventCard.scss [hunk 34]
```diff
@@ -1,29 +1,88 @@
...
+          color: $white;
+          @include flex(row, center, start);
+        }
+      }
+    }
+  }
+  .month {
+    padding: $minPadding;
+    color: $lightThemeAccept;
+    font-family: 'Robinson', sans-serif;
```
Comment (andrii-kuzmichov) [inline src/components/landing/EventCard/EventCard.scss]: use constants

File: src/components/landing/EventCard/IconsEventCard/IconsEventCard.scss [hunk 35]
```diff
@@ -12,17 +12,13 @@ $postition-icons-list-right: 75px;
 }
 
 .join-to-event__icon {
-  background: url("../../../../assets/icons/ImgJoinToEvent.svg") no-repeat
-    center;
+  background: url("../../../../assets/icons/ImgJoinToEvent.svg") no-repeat center;
```
Comment (andrii-kuzmichov) [inline src/components/landing/EventCard/IconsEventCard/IconsEventCard.scss:15]: constants

File: src/components/landing/EventMatrix/EventMatrix.js [hunk 36]
```diff
@@ -1,14 +1,13 @@
 import React from "react";
 import PropTypes from "prop-types";
 import EventCard from "../EventCard/EventCard";
-import Image from "../../../mockup-images/football-g10e9bda93_1920.jpg";
 import "./EventMatrix.scss";
 
 export default function EventMatrix({ events }) {
   return (
```
Comment (andrii-kuzmichov) [inline src/components/landing/EventMatrix/EventMatrix.js:7]: drop return

File: src/components/landing/EventsListItem/EventListCard/EventListCard.js [hunk 37]
```diff
@@ -1,44 +1,77 @@
...
+    "April",
+    "May",
+    "June",
+    "July",
+    "August",
+    "September",
+    "October",
+    "November",
+    "December",
+  ];
```
Comment (andrii-kuzmichov) [inline src/components/landing/EventsListItem/EventListCard/EventListCard.js]: Drop

File: src/components/landing/EventsListItem/EventListCard/EventListCard.js [hunk 38]
```diff
@@ -1,44 +1,77 @@
...
   const [eventImage, setEventImage] = useState(eventDefaultImage);
 
   useEffect(() => {
-    photoService.getFullEventPhoto(data?.id).then(eventFullImage => {
-      if (eventFullImage !== null && eventFullImage !== undefined) {
-        setEventImage(URL.createObjectURL(eventFullImage));
+    photoService.getPreviewEventPhoto(id).then(eventPreviewImage => {
+      if (eventPreviewImage) {
+        const image = URL.createObjectURL(eventPreviewImage);
+        setEventImage(image);
```
Comment (andrii-kuzmichov) [inline src/components/landing/EventsListItem/EventListCard/EventListCard.js:21]: move to service

File: src/components/landing/EventsListItem/EventListCard/EventListCard.js [hunk 39]
```diff
@@ -1,44 +1,77 @@
 import React, { useState, useEffect } from "react";
```
Comment (andrii-kuzmichov) [inline src/components/landing/EventsListItem/EventListCard/EventListCard.js:1]: Try to create unified component for this and above

File: src/components/landing/EventsListItem/EventListCard/EventListCard.scss [hunk 40]
```diff
@@ -1,47 +1,248 @@
...
+        width: $distance370px;
+        height: $distance200px;
+        object-fit:cover;
+      }
+      .card-item-content {
+        @include flex(column, start, space-between);
+        width: 40%;
+        .card-item-header {
+          text-transform: uppercase;
+          font-weight: bold;
```
Comment (andrii-kuzmichov) [inline src/components/landing/EventsListItem/EventListCard/EventListCard.scss]: constants

File: src/components/landing/EventsListItem/EventListCard/EventListCard.scss [hunk 41]
```diff
@@ -1,47 +1,248 @@
...
+          color: $lightThemeAccept;
+          position: relative;
+        }
+        .card-item-ovner {
+          @include flex(row, center, center);
+          width: $percent100;
+          color: $black;
+          background-color: $lightPink;
+          border-radius: 3px;
+          font-weight: bold;
```
Comment (andrii-kuzmichov) [inline src/components/landing/EventsListItem/EventListCard/EventListCard.scss]: constants

File: src/components/landing/EventsListItem/EventListCard/EventListCard.scss [hunk 42]
```diff
@@ -1,47 +1,248 @@
...
+        margin-bottom: 0;
+      }
+      .event-date {
+        @include flex(column, start, start);
+        position: absolute;
+        left: $distance30px;
+        bottom: $distance25px;
+        .day, .month {
+          color: $lightThemeAccept;
+          font-family: 'Robinson', sans-serif;
```
Comment (andrii-kuzmichov) [inline src/components/landing/EventsListItem/EventListCard/EventListCard.scss]: constants

File: src/components/landing/EventsListItem/EventListCard/EventListCard.scss [hunk 43]
```diff
@@ -1,47 +1,248 @@
...
+          height: $distance40px;
+        }
+        .day {
+          text-shadow: 0 0 $distance20px $white;
+          margin-bottom: $distance15px;
+          font-weight: bold;
```
Comment (andrii-kuzmichov) [inline src/components/landing/EventsListItem/EventListCard/EventListCard.scss]: constants

File: src/components/landing/EventsListItem/EventListCard/EventListCard.scss [hunk 44]
```diff
@@ -1,47 +1,248 @@
...
+        position: absolute;
+        left: $distance30px;
+        bottom: $distance25px;
+        .day, .month {
+          color: $lightThemeAccept;
+          font-family: 'Robinson', sans-serif;
+          height: $distance40px;
+        }
+        .day {
+          text-shadow: 0 0 $distance20px $white;
```
Comment (andrii-kuzmichov) [inline src/components/landing/EventsListItem/EventListCard/EventListCard.scss]: constants

File: src/components/landing/EventsListItem/EventListCard/EventListCard.scss [hunk 45]
```diff
@@ -1,47 +1,248 @@
...
+          margin-bottom: $distance15px;
+          font-weight: bold;
+          font-size: $distance100px;
+          line-height: $distance30px;
+        }
+        .month {
+          text-transform: uppercase;
+          font-size: $distance45px;
```
Comment (andrii-kuzmichov) [inline src/components/landing/EventsListItem/EventListCard/EventListCard.scss]: typo

File: src/components/landing/EventsListItem/EventListCard/EventListCard.scss [hunk 46]
```diff
@@ -1,47 +1,248 @@
...
+          .card-image {
+            min-width: $distance280px;
+            max-height: $distance150px;
+          }
+        }
+        .card-item-content {    
+          height: $distance200px;
+          justify-content: flex-start;
+          .card-item-header {
+            font-size: 12px;
```
Comment (andrii-kuzmichov) [inline src/components/landing/EventsListItem/EventListCard/EventListCard.scss:135]: constants

File: src/components/landing/EventsListItem/EventListCard/EventListCard.scss [hunk 47]
```diff
@@ -1,47 +1,248 @@
...
+            padding-bottom: $distance10px;
+          }
+          .card-item-description {
+            font-size: 12px;
```
Comment (andrii-kuzmichov) [inline src/components/landing/EventsListItem/EventListCard/EventListCard.scss:139]: constants

File: src/components/landing/EventsListItem/EventListCard/EventListCard.scss [hunk 48]
```diff
@@ -1,47 +1,248 @@
...
+        .event-date {
+          position: absolute;
+          left: $distance30px;
+          bottom: $distance25px;
+          display: flex;
+          flex-direction: column;
+          
+          .day, .month {
+            color: $lightThemeAccept;
+            font-family: 'Robinson', sans-serif;
```
Comment (andrii-kuzmichov) [inline src/components/landing/EventsListItem/EventListCard/EventListCard.scss]: constants

File: src/components/landing/EventsListItem/EventListCard/EventListCard.scss [hunk 49]
```diff
@@ -1,47 +1,248 @@
...
+            height: $distance40px;
+          }
+          .day {
+            text-shadow: 0 0 $distance20px $white;
+            margin-bottom: $distance10px;
+            font-weight: bold;
```
Comment (andrii-kuzmichov) [inline src/components/landing/EventsListItem/EventListCard/EventListCard.scss]: constants

File: src/components/landing/EventsListItem/EventListCard/EventListCard.scss [hunk 50]
```diff
@@ -1,47 +1,248 @@
...
+          display: flex;
+          flex-direction: column;
+          
+          .day, .month {
+            color: $lightThemeAccept;
+            font-family: 'Robinson', sans-serif;
+            height: $distance40px;
+          }
+          .day {
+            text-shadow: 0 0 $distance20px $white;
```
Comment (andrii-kuzmichov) [inline src/components/landing/EventsListItem/EventListCard/EventListCard.scss]: constants

File: src/components/landing/EventsListItem/EventListCard/EventListCard.scss [hunk 51]
```diff
@@ -1,47 +1,248 @@
...
+            margin-bottom: $distance10px;
+            font-weight: bold;
+            font-size: $distance80px;
```
Comment (andrii-kuzmichov) [inline src/components/landing/EventsListItem/EventListCard/EventListCard.scss]: typo

File: src/components/landing/EventsListItem/EventListCard/EventListCard.scss [hunk 52]
```diff
@@ -1,47 +1,248 @@
...
+            line-height: $distance30px;
+          }
+          .month {
+            text-transform: uppercase;
+            font-size: 40px;
```
Comment (andrii-kuzmichov) [inline src/components/landing/EventsListItem/EventListCard/EventListCard.scss:171]: constants

File: src/components/landing/EventsListItem/EventListCard/EventListCard.scss [hunk 53]
```diff
@@ -1,47 +1,248 @@
...
+            display: none;
+          }
+        }
+      }
+      .card-item-bottom-row {
+        padding-bottom: $distance10px;
+        .event-date {
+          left: $distance20px;
+          .day {
+            text-shadow: 0 0 $distance10px $lightThemeHeader;
```
Comment (andrii-kuzmichov) [inline src/components/landing/EventsListItem/EventListCard/EventListCard.scss]: constants

File: src/components/landing/EventsListItem/EventListCard/EventListCard.scss [hunk 54]
```diff
@@ -1,47 +1,248 @@
...
+            margin-bottom: $distance10px;
+            font-weight: bold;
```
Comment (andrii-kuzmichov) [inline src/components/landing/EventsListItem/EventListCard/EventListCard.scss]: constants

File: src/components/landing/EventsListItem/EventListCard/EventListCard.scss [hunk 55]
```diff
@@ -1,47 +1,248 @@
...
+            font-size: $distance70px;
```
Comment (andrii-kuzmichov) [inline src/components/landing/EventsListItem/EventListCard/EventListCard.scss]: typo

File: src/components/landing/EventsListItem/EventListCard/EventListCard.scss [hunk 56]
```diff
@@ -1,47 +1,248 @@
...
+            line-height: $distance70px;
+          }
+          .month {
+            text-transform: uppercase;
+            font-size: $distance40px;
```
Comment (andrii-kuzmichov) [inline src/components/landing/EventsListItem/EventListCard/EventListCard.scss]: typo

File: src/components/landing/EventsListItem/EventsListItem.js [hunk 57]
```diff
@@ -3,7 +3,7 @@ import PropTypes from "prop-types";
 import EventListCard from "./EventListCard/EventListCard";
 
 const EventsListItem = ({ events }) =>
-  events.map(event => <EventListCard key={event.id} event={event} />);
+  events.map(event => <EventListCard key={event.id + 5} event={event} />);
```
Comment (andrii-kuzmichov) [inline src/components/landing/EventsListItem/EventsListItem.js:6]: magic number

File: src/components/landing/SliderForEvents/SliderForEvents.scss [hunk 58]
```diff
@@ -7,38 +7,15 @@ $slick-prev-after-translate: translate(-75%, 50%);
...
 .slick-prev:before,
 .slick-prev:after,
 .slick-next:before,
 .slick-next:after {
   content: "";
   position: absolute;
   height: 2px;
-  width: 25px;
-  background: $black;
+  width: $distance25px;
```
Comment (andrii-kuzmichov) [inline src/components/landing/SliderForEvents/SliderForEvents.scss]: bad name

File: src/styles/_variables.scss [hunk 59]
```diff
@@ -1,14 +1,17 @@
...
 
 $mainFont: "Montserrat", sans-serif;
 
 $black: #000;
 $white: #fff;
+$red: #cd2121;
 $gray: #f2f2f2;
 $lightThemeBackground: #fff;
 $lightThemeAccept: #6c488a;
+$lightPink: #E4D5EE;
```
Comment (andrii-kuzmichov) [inline src/styles/_variables.scss]: use small case

Comment (andrii-kuzmichov) [inline src/styles/_variables.scss]: Maybe should be used in colors.scss

**Review by andrii-kuzmichov — COMMENTED** [review]
Drop all specific measures

**Review by MelenkoOleksandr — DISMISSED** [review]

**Review by Danyil-man — DISMISSED** [review]

**Review by andrii-kuzmichov — DISMISSED** [review]

**Review by andrii-kuzmichov — DISMISSED** [review]

**Review by andrii-kuzmichov — DISMISSED** [review]

**Review by andrii-kuzmichov — APPROVED** [review]

**Pull request merged**